package.json

```diff
@@ -1,21 +1,17 @@
 {
-  "name": "actyx-advanced-tutorial-example-app",
+  "name": "actyx-andvanced-tutorial",
```

Comment:
we should call this advanced example. It does not have any part of a tutorial and is build as example

Reply:
done